pysixdesk/lib/dbadaptor.py

```diff
@@ -29,7 +29,7 @@ def create_table(self, conn, name, columns, keys, recreate):
             auto_keys = keys['autoincrement']
             for ky in auto_keys:
                 columns[ky] = columns[ky] + ' AUTO_INCREMENT'
-        col = [' '.join(map(str, i)) for i in columns.items()]
+        col = [f'`{k}` {v}' for k, v in columns.items()]
```

Comment:
Yes these changes, where I add the \`, was because there was no sql sanitizing, meaning strings with particular chars would completely break the sql syntax. Adding the \` fixes it, but it is quite cumbersome.